scripts/remove

```diff
@@ -33,16 +33,20 @@ source /usr/share/yunohost/helpers
 app=$YNH_APP_INSTANCE_NAME
 domain=$(ynh_app_setting_get $app domain)
 
+service_name="ynh-vpnclient"
+service_checker_name=$service_name"-checker"
+
 #=================================================
-# The End
-ynh_systemctl stop ynh-vpnclient-checker.service
-systemctl disable ynh-vpnclient-checker.service
-ynh_systemctl stop ynh-vpnclient-checker.timer && sleep 1
-systemctl disable ynh-vpnclient-checker.timer
-ynh_systemctl stop ynh-vpnclient
-systemctl disable ynh-vpnclient
-yunohost service remove ynh-vpnclient
-for FILE in $(ls /etc/systemd/system/ynh-vpnclient* /usr/local/bin/ynh-vpnclient* /tmp/.ynh-vpnclient-*)
+
+# Stop and remove yunohost services
+yunohost service stop $service_checker_name.service
+yunohost service disable $service_checker_name.service
+yunohost service stop $service_checker_name.timer && sleep 1
+yunohost service disable $service_checker_name.timer
+yunohost service stop $service_name
+yunohost service disable $service_name
+yunohost service remove $service_name
```

Comment:
I think we also have to remove $service_checker_name.timer and $service_checker_name.service from yunohost (otherwise if you remove/reinstall vpnclient, the old client will still be there and conflict)